disnake/threads.py

```diff
@@ -394,6 +416,25 @@ def is_pinned(self) -> bool:
         """
         return self.flags.pinned
 
+    # TODO: `applied_tags` instead of `tags`?
+    @property
+    def tags(self) -> List[ThreadTag]:
+        """List[:class:`ThreadTag`]: The tags currently applied to this thread.
+        Only applicable to threads in :class:`ForumChannel`\\s.
+
+        The returned list can be mutated, and will not change internal state.
```

Comment:
I don't think this line is necessary.

Reply:
The idea is that we guarantee that this is safe:
```py
tags = thread.applied_tags
tags.append(new_tag)
await thread.edit(applied_tags=tags)
```
But we could just not do that and instead expect people to copy or use `[*thread.applied_tags, new_tag]`, which would also be fine. Thoughts?

Reply:
actually, nevermind. updated the docs in https://github.com/DisnakeDev/disnake/pull/726/commits/95f312d25cb81ca4465d3436a1652a9b8e408755